ember-cli-build.js

```diff
@@ -103,23 +103,9 @@ function routeRecognizer() {
 }
 
 function router() {
-  // TODO upstream this to router.js and publish on npm
-  return new Rollup('bower_components/router.js/lib', {
+  return new Rollup(path.resolve(path.dirname(require.resolve('router_js')), '../lib'), {
```

Comment:
Confirm we don't want the `main`, but maybe `'jsnext:main'` or [`directories.lib`](https://docs.npmjs.com/files/package.json#directorieslib).